consistency-tests/__tests__/clickables.test.tsx

```diff
@@ -184,10 +190,18 @@ describe.each`
 `("$name", ({Component, name, hasTabIndex}: any) => {
     test("has expected existence of tabIndex", async () => {
         // Arrange
+        // @ts-expect-error [FEI-5019] - TS2790 - The operand of a 'delete' operator must be optional.
+        delete window.location;
+        // @ts-expect-error [FEI-5019] - TS2740 - Type '{ assign: Mock<any, any, any>; }' is missing the following properties from type 'Location': ancestorOrigins, hash, host, hostname, and 8 more.
+        window.location = {assign: jest.fn(), origin: "http://localhost"};
 
         // Act
         render(
-            <Component onClick={jest.fn()} testId="clickable-component-test-id">
+            <Component
+                onClick={jest.fn()}
+                testId="clickable-component-test-id"
+                href="#" // Required for Link component
```

Comment:
I don't love putting `href` on all of these variants since they probably aren't all links. But I didn't want to hack on anything too involved, either!

Reply:
If we set the `href` prop, the rendered component will always be a link! It looks like this test case is for other interactive components too. Maybe we can conditionally set `href`? 

Also, do you know how the test was passing before without setting `href`?